PersonApi/V1/Factories/PersonSnsFactory.cs

```diff
@@ -15,10 +15,31 @@ public PersonSns Create(Person person, Token token)
                 DateTime = DateTime.UtcNow,
                 EntityId = person.Id,
                 Id = Guid.NewGuid(),
-                EventType = Constants.EVENTTYPE,
-                Version = Constants.V1_VERSION,
-                SourceDomain = Constants.SOURCE_DOMAIN,
-                SourceSystem = Constants.SOURCE_SYSTEM,
+                EventType = CreateEventConstants.EVENTTYPE,
+                Version = CreateEventConstants.V1VERSION,
+                SourceDomain = CreateEventConstants.SOURCEDOMAIN,
+                SourceSystem = CreateEventConstants.SOURCESYSTEM,
+                User = new User
+                {
+                    Id = Guid.NewGuid(),
+                    Name = token.Name,
+                    Email = token.Email
+                }
+            };
+        }
+
+        public PersonSns Update(Person person, Token token)
+        {
+            return new PersonSns
+            {
+                CorrelationId = Guid.NewGuid(),
+                DateTime = DateTime.UtcNow,
+                EntityId = person.Id,
+                Id = Guid.NewGuid(),
+                EventType = UpdatePersonConstants.EVENTTYPE,
+                Version = UpdatePersonConstants.V1VERSION,
+                SourceDomain = UpdatePersonConstants.SOURCEDOMAIN,
+                SourceSystem = UpdatePersonConstants.SOURCESYSTEM,
```

Comment:
You dont need to have different update and create constants. A single place for constants is fine.

Reply:
But the value for the EventType is different, that's why you need different constants